### PR TITLE
Implement Fetch and List purchase order routes

### PR DESCRIPTION
**Pull request by dplumb94**

### CODE REVIEW

File: sdk/src/rest_api/actix_web_3/routes/purchase_orders.rs [hunk 1]
```diff
@@ -12,36 +12,88 @@
...
-    _query_paging: web::Query<QueryPaging>,
+    state: web::Data<StoreState>,
+    query_org_id: web::Query<QueryOrgId>,
+    query_service_id: web::Query<QueryServiceId>,
+    query_paging: web::Query<QueryPaging>,
     version: ProtocolVersion,
     _: AcceptServiceIdParam,
 ) -> HttpResponse {
     match version {
+<<<<<<< HEAD
```
Comment (dnewh) [inline sdk/src/rest_api/actix_web_3/routes/purchase_orders.rs:40]: looks like some merge conflicts got committed

File: sdk/src/rest_api/actix_web_3/routes/purchase_orders.rs [hunk 2]
```diff
@@ -12,36 +12,88 @@
...
-    _uuid: web::Path<String>,
-    _query: web::Query<QueryServiceId>,
+    state: web::Data<StoreState>,
+    uuid: web::Path<String>,
+    query_service_id: web::Query<QueryServiceId>,
     version: ProtocolVersion,
     _: AcceptServiceIdParam,
 ) -> HttpResponse {
     match version {
+<<<<<<< HEAD
```
Comment (dnewh) [inline sdk/src/rest_api/actix_web_3/routes/purchase_orders.rs:77]: here too

**Review by dnewh — CHANGES_REQUESTED** [review]

File: sdk/src/rest_api/actix_web_3/routes/purchase_orders.rs (hunk 1 above)
Comment (dplumb94) [reply sdk/src/rest_api/actix_web_3/routes/purchase_orders.rs]: fixed

Comment (dplumb94) [reply sdk/src/rest_api/actix_web_3/routes/purchase_orders.rs]: forgot to save file before committing it heh

**Review by dnewh — DISMISSED** [review]

File: sdk/src/rest_api/resources/purchase_order/v1/payloads.rs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    #[serde(default)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub version_numbers: Option<Vec<String>>,
+    pub is_closed: bool,
+}
+
+impl From<PurchaseOrder> for PurchaseOrderSlice {
+    fn from(purchase_order: PurchaseOrder) -> Self {
+        Self {
+            uuid: purchase_order.uuid.clone(),
```
Comment (vaporos) [inline sdk/src/rest_api/resources/purchase_order/v1/payloads.rs:33]: Are these clone() needed here since purchase_order is consumed?

Comment (dplumb94) [reply sdk/src/rest_api/resources/purchase_order/v1/payloads.rs]: They were not necessary. Updated.

**Review by vaporos — APPROVED** [review]

**Review by dnewh — APPROVED** [review]

**Pull request merged**